packages/core/src/components/EmptyState/EmptyState.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button } from '@material-ui/core';
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { EmptyState } from './EmptyState';
+
+export default {
+  title: 'EmptyState',
+  component: EmptyState,
+};
+
+const containerStyle = { width: '100%', height: '100vh' };
+
+export const MissingAnnotation = () => (
+  <div style={containerStyle}>
+    <EmptyState
+      type="missingAnnotation"
+      title="Your plugin is missing an annotation"
+      description="Add a descripton here."
```

Comment:
description* :grin:

Btw, something like https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker can save you from a lot of these :>

Reply:
Thank you